package.json

```diff
@@ -102,6 +102,7 @@
     "react-autosuggest": "^9.3.3",
     "react-avatar": "^2.5.1",
     "react-color": "^2.13.8",
+    "react-container-query": "^0.11.0",
```

Comment:
I don't see this imported anywhere?

Reply:
@zenweasel This is used in the CatalogGrid component. According to the "Verify Peer Dependencies" section [here](https://stoic-hodgkin-c0179e.netlify.com/), you need to install the the component lib's peer dependencies in your app. If I remove line 105 I get an error on startup:

```
Unable to resolve some modules:

  "react-container-query" in                  
/Users/dan/reaction/reaction/node_modules/@reactioncommerce/components/CatalogGrid/v1/CatalogGrid.js
(web.browser.legacy)
```